src/applications/hca/containers/IDPage.jsx

```diff
@@ -49,12 +63,73 @@ class IDPage extends React.Component {
     this.props.toggleLoginModal(true);
   };
 
+  renderServerErrorAlert = () => (
+    <AlertBox
+      isVisible
+      status="error"
+      headline="Server Error"
+      content={
+        <>
+          <p>
+            We’re sorry for the interruption, but we have encountered an error.
+            Please try again later.
+          </p>
+        </>
+      }
+    />
+  );
+
+  renderContinueButton = () => (
+    <LoadingButton
+      isLoading={this.props.isSubmittingIDForm}
+      disabled={false}
+      type="submit"
+      /* to override the `width: 100%` given to SchemaForm submit buttons */
+      style={{ width: 'auto' }}
+    >
+      Continue to the Application
+      <span className="button-icon">&nbsp;»</span>
+    </LoadingButton>
+  );
+
+  renderLoginRequiredAlert = () => (
+    <>
+      <AlertBox
```

Comment:
You could move them into their own stateless functional components. They don't have to be separate files and you could reference them as something like `loginRequired && <LoginRequiredAlert />`.